workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -47,6 +63,16 @@ private void OnDestroy()
             Dispose();
         }
 
+        /// <summary>
+        ///     Connects a worker to the SpatialOS runtime.
```

Comment:
asynchronously
Also just realised this should be named ConnectAsync. Will try to fix that asap